content/virtualization.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Virtualization on Linux
+description: Create a VM on Linux with GNOME Boxes or Virtual Machine Manager
+keywords:
+  - VM
+  - libvirt
+  - GNOME Boxes
+  - virt-manager
+  - Virtual Machine Manager
+  - VMM
+  - VirtualBox
+  - VMWare
+  - Virtual Machine
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+![GNOME Boxes with VMs](/images/gnome-boxes/boxes-full.png)
+
+This is a quick tutorial on running virtual machines, or VMs, on Linux. Virtual machines allow you to run other operating systems from within your primary operating systems. This is a great way to run applications that require older software, run applications that require an OS other than Linux, or to try out other operating systems.
```

Comment:
Minor conciseness - If you change the title to "Creating Virtual Machines on Linux" you can get rid of this first sentence. "Virtualization in Linux" can mean a few things.

Reply:
Fair point about the title, I'll update that :+1: